### PR TITLE
Add length validation

### PR DESCRIPTION
**Pull request by aroltsch**
Add length validation for the title of certifying official. 

image

### CODE REVIEW

File: client/app/certification/SignAndCertify.jsx [hunk 1]
```diff
@@ -35,7 +35,8 @@ const ERRORS = {
   certifyingOfficialName: 'Please enter the name of the certifying official (usually your name).',
   certifyingOfficialTitle: 'Please enter the title of the certifying official.',
   certificationDate: "Please enter today's date.",
-  certifyingOfficialNameLength: 'Maximum length of certifying official\'s name reached.'
+  certifyingOfficialNameLength: 'Maximum length of certifying official\'s name reached.',
+  certifyingOfficialTitleOtherLength: 'Maximum length of certifying official\'s title reached.'
```
Comment (sharonwarner) [inline client/app/certification/SignAndCertify.jsx]: I think we should either tell the user how many characters they're allowed to input or have the message go away when they're back under the limit.

Comment (aroltsch) [inline client/app/certification/SignAndCertify.jsx]: @amprokop what are your thoughts on this?

Comment (amprokop) [inline client/app/certification/SignAndCertify.jsx]: I'd go with whatever @shellicious and designers say here!

Comment (shellicious) [inline client/app/certification/SignAndCertify.jsx]: The error message should say: "Please enter less than ____ characters in the field below." if the message is too long. @sharonwarner @amprokop @aroltsch

Comment (aroltsch) [inline client/app/certification/SignAndCertify.jsx]: image

**Pull request merged**